### PR TITLE
Toast Message component has been added

### PR DESCRIPTION
**Pull request by Dorjay-Yolmo**
## Summary
Added new Toast message component using Material Snackbar

### CODE REVIEW

File: src/components/UI/ToastMessage/ToastMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  severity: Severity;
+  message: String;
+  seconds: number;
+  handleClose: Function;
+}
+
+// Since attribute severity can only have 5 values,
+type Severity = 'error' | 'success' | 'info' | 'warning' | undefined;
+
+function Alert(props: AlertProps) {
```
Comment (kurund) [inline src/components/UI/ToastMessage/ToastMessage.tsx:17]: Please use ES6 syntax here.

File: src/components/UI/ToastMessage/ToastMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Snackbar, Button, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
```
Comment (kurund) [inline src/components/UI/ToastMessage/ToastMessage.tsx:3]: I don't see this is been used anywhere. Do we need this?

Comment (Dorjay-Yolmo) [reply src/components/UI/ToastMessage/ToastMessage.tsx:3]: I was planning on using them. Will remove it and checkin.

**Comment by kurund** [comment]
@Dorjay-Yolmo Can you please add test cases for this component.

**Pull request merged**